reactivesocket-core/src/main/java/io/reactivesocket/internal/Responder.java

```diff
@@ -483,16 +485,16 @@ private void cleanup() {
                 }
 
             };
-            synchronized(Responder.this) {
+            synchronized(this) {
```

Comment:
`Responder.this` is correct here, `this` represents the `new Subscriber` line 417.